.eslintrc

```diff
@@ -9,6 +9,7 @@
     }
   },
   "rules": {
-    "no-invalid-this": 0
+    "no-invalid-this": 0,
+    "react/no-deprecated": 0
```

Comment:
Shouldn't we update react-carousel to not use deprecated methods?

Reply:
We should. I could not do it fast. It's on our TODO list.